uvicorn/protocols/http/h11_impl.py

```diff
@@ -214,7 +205,7 @@ def handle_events(self) -> None:
                 self.flow.pause_reading()
                 break
 
-            elif event_type is h11.Request:
+            elif isinstance(event, h11.Request):
```

Comment:
How more expensive are those compared to `is`?

Reply:
Well... I think we care far more about correctness than performance on `h11`.

Reply:
I recall them being concerned about the cost of doing that check when I made a similar change in a hot loop in h11. I think it's okay though. Will this break Uvicorn for users that don't have h11 from git or did this change land in an earlier release?

Reply:
Let me check

Reply:
It does work. 👍